Report/sections/1_analysis/Behavior.tex

```diff
@@ -0,0 +1,19 @@
+\chapter{Behavior}\label{chap:behavior}
+
+As concluded in the last chapter, learning how to program for the first time is hard.
+Therefore, a natural next question would be to ask how one could make it easier.
```

Comment:
The previous statement was not a question so saying "a natural **next** question" seems weird.